src/main/java/com/recordit/server/service/RecordService.java

```diff
@@ -124,4 +131,42 @@ public RecordDetailResponseDto getDetailRecord(Long recordId) {
 				.imageUrls(imageUrls)
 				.build();
 	}
+
+	@Transactional(readOnly = true)
+	public TodayWriteRecordResponseDto getTodayWriteRecord() {
+
+		Long userIdBySession = sessionUtil.findUserIdBySession();
+		log.info("세션에서 찾은 사용자 ID : {}", userIdBySession);
+
+		Member member = memberRepository.findById(userIdBySession)
+				.orElseThrow(() -> new MemberNotFoundException("회원 정보를 찾을 수 없습니다."));
+
+		Optional<Record> optionalRecord = recordRepository.findTopByWriterAndCreatedAtBetweenOrderByCreatedAtDesc(
+				member, LocalDateTime.of(LocalDate.now(), LocalTime.MIN),
+				LocalDateTime.of(LocalDate.now(), LocalTime.MAX));
+
+		if (!optionalRecord.isPresent()) {
+			return TodayWriteRecordResponseDto.builder().build();
+		}
+
+		Record findRecord = optionalRecord.get();
+		Long commentCount = commentRepository.countByWriter(member);
+
+		log.info("오늘 작성한 가장 최신의 레코드 : {}", findRecord.getTitle());
+		log.info("오늘 작성한 가장 최신 레코드의 댓글 갯수 : {}", commentCount);
+
+		return TodayWriteRecordResponseDto.builder()
+				.todayWriteRecordDto(TodayWriteRecordDto.builder()
+						.commentCount(commentCount)
+						.categoryName(findRecord.getRecordCategory().getName())
+						.title(findRecord.getTitle())
+						.createdAt(findRecord.getCreatedAt())
+						.colorName(findRecord.getRecordColor().getName())
+						.colorHexCode(findRecord.getRecordColor().getHexCode())
+						.iconName(findRecord.getRecordIcon().getName())
+						.build())
+				.build();
```

Comment:
```suggestion
		return TodayWriteRecordResponseDto.builder()
				.commentCount(commentCount)
				.categoryName(findRecord.getRecordCategory().getName())
				.title(findRecord.getTitle())
				.createdAt(findRecord.getCreatedAt())
				.colorName(findRecord.getRecordColor().getName())
				.colorHexCode(findRecord.getRecordColor().getHexCode())
				.iconName(findRecord.getRecordIcon().getName())
				.build();
```
한개의 dto안에 또 다른 dto를 넣을 필요없이 한번에 하는게 좋아보임니다